widget/popup_menu.go

```diff
@@ -67,6 +67,15 @@ func (p *PopUpMenu) Move(pos fyne.Position) {
 	p.Base.Move(p.adjustedPosition(pos, p.Size()))
 }
 
+// Refresh ensures the menu updates to reflect data changes.
+//
+// Implements: fyne.Widget
+func (p *PopUpMenu) Refresh() {
+	p.updateItems()
```

Comment:
I think this is unnecessary as p.Menu.Refresh() will already update their items?

Reply:
Ah yes good point. Not quite that simple, but duplication nonetheless. Fix inbound.